alces-flight-www/metadata.yml

```diff
@@ -94,6 +95,14 @@ component-base:
       _install_nginx_conf document-root server-http
       firewall_rules+=(http)
       echo "cw_CLUSTER_FIREWALL_rule_http=\"INPUT -p tcp --dport ${cw_ALCES_FLIGHT_WWW_http_port:-80} -j ACCEPT\"" >> "${cw_ROOT}"/etc/cluster-firewall/static.d/alces-flight-www.rc
+
+      files_load_config config config/cluster
+      prv_net=$(network_get_iface_network "$cw_CLUSTER_iface")
```

Comment:
Does `$cw_CLUSTER_iface` always exist? Might be worth using ` ${cw_CLUSTER_iface:-$(network_get_first_iface)}` which we use elsewhere.